tests/tasks/setup.yml

```
@@ -14,3 +14,20 @@
   package:
     name: "{{ __ssh_test_packages }}"
     state: present
+
+- name: Define common variables
+  ansible.builtin.set_fact:
+    main_ssh_config: /etc/ssh/ssh_config
+    main_ssh_config_name: ssh_config
+    main_ssh_config_path: /etc/ssh/
+
+- name: Define specific variables
+  ansible.builtin.set_fact:
+    main_ssh_config: /etc/ssh/ssh_config.d/00-ansible.conf
+    main_ssh_config_name: 00-ansible.conf
+    main_ssh_config_path: /etc/ssh/ssh_config.d/
+  when:
+    - (ansible_facts['os_family'] == 'RedHat'
+        and ansible_facts['distribution_major_version']|int >= 8) or
```

```suggestion
        and ansible_facts['distribution_major_version'] | int >= 8) or
```
I guess the ansible-lint github action still hasn't updated to include the fix for checking jinja in multiline `when`